### ISSUE TITLE
Issue with Tensorflow installation

### ISSUE THREAD

**Issue by Paasu** [neuropoly/axondeepseg#346]
**Describe the bug**
After performing a manual update to my axondeepseg, (I just downloaded the zip of GitHub), it requests that I install Tensorflow 2.2 everytime I try to train but when I try the install fails; it requires 2 dependencies (update to setuptools and an update to another depedency) l which when I try update, it doesn't actually update even though it says update successful. I'm not well versed in linux so I apologize if this is not the correct place to ask. My guess is that it is just an issue with my OS, I think it might be best to clean install ubuntu. What do you guys think?

Additionally, the way I installed ADS was by just downloading the zip, manually installing and running the training file. It just dawned on me that it might be better to install by using git clone. However doing that gives me same issue as #345. 

**Desktop (please complete the following information):**
 - OS: [OpenSUSE]
 - Version [15.1]
AXONDEEPSEG VERSION - LATEST VERSION AVAILABLE ON PYTHON 3.6

**Comment by Paasu** [comment]
@jcohenadad 
I have not done this in a conda environment, just in linux environment. Should I install in a linux environment?

Here is the terminal log and training.py:
[training.zip](https://github.com/neuropoly/axondeepseg/files/5184079/training.zip)

**Comment by jcohenadad** [comment]
> I have not done this in a conda environment, just in linux environment. Should I install in a linux environment?

i'm confused-- you said you did it in linux environment, so why are you asking if you should install it in linux env? (since you've already done that).

please do it as instructed in our documentation, with miniconda, and copy/paste entire output.

**Comment by Paasu** [comment]
@jcohenadad 
Maybe I typed incorrectly when I tried installing it before but it seems to work now, thanks!
Here I'll outline the problem and solution in case someone follows in my stupidity.

**Problem**:
I installed AxonDeepSeg by downloading the zip, extracting it and attempting to train my data.
The problem was at the time that I didn't have the dependencies to run the training, so I manually installed them using pip install xyz based on what the terminal told me I was missing until finally the training ran. Since I just downloaded the zip for ADS, the updates I performed were also manual; I would re-download the zip and move in my training files into that new ADS folder and test the training. But at one point it requested that I install TensorFlow even though I already had it installed, and trying to install it wasn't working.

**Solution**:
Use the installation method outlined in the documentation, do it in a conda environment, use git clone and install and attempt training.

Thanks again for helping me with the issue!
